docs/features/integration/config_as_code.rst

```diff
@@ -0,0 +1,68 @@
+..
+   # *******************************************************************************
+   # Copyright (c) 2024 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+Configuration as code
+#####################
+
+Documentation
+*************
+
+Description
+===========
+
+Score project uses GitHub for hosting source code as well as GitHub
+Actions for all CI/CD task. The configuration of the organization and
+repositories is managed by otterdog, a config-as-code solution. GitHub
+provides several approaches to set up repositories. In order to provide
+unified experience among all the Score repositories, the following rules
+should be followed.
+
+For in depth description of the properties and the relations, please
+refer to the otterdog documentation.
+
+Current setup can be viewed in the dashboard:
+https://otterdog.eclipse.org/projects/automotive.score
+
+Automatically generated documentation for eclipse-score otterdog
+configuration is here: https://eclipse-score.github.io/.eclipsefdn/
+
+All changes to the configuration should be handled by pull requests in
+https://github.com/eclipse-score/.eclipsefdn/
+
+Organization settings
+---------------------
+
+``default_branch_name`` - the default branch name for newly created
```

Comment:
Could you make a bullet points out of it?

Reply:
Fixed.